testing/x/replay.go

```diff
@@ -86,6 +90,10 @@ import (
 //
 // Replay does not assume that the provider is a bridged provider and can be generally useful.
 func Replay(t *testing.T, server pulumirpc.ResourceProviderServer, jsonLog string) {
+	ReplayWithOpts(t, server, jsonLog, ReplayOptions{})
+}
+
+func ReplayWithOpts(t *testing.T, server pulumirpc.ResourceProviderServer, jsonLog string, opts ReplayOptions) {
```

Comment:
Instead of `ReplayWithOpts`, it is more idiomatic to use

```go
func Replay(t *testing.T, server pulumirpc.ResourceProviderServer, jsonLog string, opts ...ReplayOption) { ... }

type ReplayOption func(*replayOption)
```

For example:

https://github.com/pulumi/pulumi-terraform-bridge/blob/fe7b24d6a9128e06d8c281bd0f2f089676abadc7/pkg/tfbridge/main.go#L30-L32